components/esp8266/include/esp8266/ets_sys.h

```diff
@@ -38,6 +38,7 @@
 
 extern char NMIIrqIsOn;
 extern uint32_t WDEV_INTEREST_EVENT;
+extern void vPortEnterCritical();
```

Comment:
This shouldn't be here and is not related to the fix in the pull request.

Reply:
Sorry about that, I made another commit after the PR.
Think Espressif can reject this PR.